### ISSUE TITLE
Log frequencies to a file or graph

### ISSUE THREAD

**Issue by hari-rangarajan** [gillesdegottex/fmit#48]
Feature request: Instead of notes with a error, would it be possible to just graph the "raw" frequencies.
Thank you!

**Comment by gillesdegottex** [comment]
Aren't you looking for something similar to the DFT view ?

**Comment by hari-rangarajan** [comment]
Yes, but a little bit more: DFT (maybe just the fundamental frequency) plotted over time.

**Comment by gillesdegottex** [comment]
Ah, so that's a real-time spectrogram :)
Yes, I'm preparing one for another software and I planned to include it in FMIT.
However, I'm running out of time these months, so I really don't know when I'll be able to do this.
